frontend/smart-kickers-game/src/apis/Game.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+export async function resetGame() {
+  try {
+    let result = await axios.post(`http://localhost:3000/reset`);
```

Comment:
```suggestion
    let result = await axios.put(`http://localhost:3000/reset`);
```
We have changed POST method to PUT on backend

Reply:
Just a question, why resetting the game is a PUT? As I'm reading in your backend, it doesn't create new or update existing resource:
```
func (g *game) ResetScore() {
	g.score.BlueScore = 0
	g.score.WhiteScore = 0
}
```

Reply:
We thought PUT would be appropriate here because it updates the score. Do you have any suggestions which other method could we use instead?

Reply:
PUT suggests that _something_ will be replaced with the contents sent in request body, but here we don't send anything. As we can read on https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods:

```
The POST method submits an entity to the specified resource, often causing a change in state or side effects on the server.
```

`ResetScore` causes a side effect on the server, so I'd go with that. TBH I've always seen  `POST` as a catch-all method if anything else matches...